### ISSUE TITLE
error when installing for go from source.

### ISSUE THREAD

**Issue by lolbinarycat** [tensorflow/tensorflow#42632]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
Linux Debian 10
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
N/A
- TensorFlow installed from (source or binary):
source
- TensorFlow version:
commit 2d0592a000989c28e94dbc6efabe6c3be54762e1
- Python version:
Python 3.7.3
- Installed using virtualenv? pip? conda?:
comiled from source, following [this](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/go/README.md)
- Bazel version (if compiling from source):
bazel 3.1.0
- GCC/Compiler version (if compiling from source):
gcc (Debian 8.3.0-6) 8.3.0
- CUDA/cuDNN version:
no idea
- GPU model and memory:

**Describe the problem**
on step 4 of the instructions (`go generate github.com/tensorflow/tensorflow/tensorflow/go/op`) I got:
```
google/protobuf/any.proto: File not found.
google/protobuf/duration.proto: File not found.
tensorflow/core/protobuf/autotuning.proto:10:1: Import "google/protobuf/any.proto" was not found or had errors.
tensorflow/core/protobuf/autotuning.proto:11:1: Import "google/protobuf/duration.proto" was not found or had errors.
tensorflow/core/protobuf/autotuning.proto:61:3: "google.protobuf.Duration" is not defined.
tensorflow/core/protobuf/autotuning.proto:74:3: "google.protobuf.Any" is not defined.
../genop/generate.go:19: running "bash": exit status 1
op/generate.go:17: running "go": exit status 1
```
**Provide the exact sequence of commands / steps that you executed before running into the problem**
```
go get -d github.com/tensorflow/tensorflow/tensorflow/go

cd ${GOPATH}/src/github.com/tensorflow/tensorflow
./configure
bazel build -c opt //tensorflow:libtensorflow.so

sudo cp ${GOPATH}/src/github.com/tensorflow/tensorflow/bazel-bin/tensorflow/libtensorflow.so /usr/local/lib
sudo cp ${GOPATH}/src/github.com/tensorflow/tensorflow/bazel-bin/tensorflow/libtensorflow_framework.so.2 /usr/local/lib/libtensorflow_framework.so

go generate github.com/tensorflow/tensorflow/tensorflow/go/op
```

**Any other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by Saduf2019** [comment]
@lolbinarycat 
Can you try "go get github.com/tensorflow/tensorflow/tensorflow/go" and let us know, please refer to: [link](https://github.com/tensorflow/tensorflow/issues/23681)
#34580 [link](https://stackoverflow.com/questions/60631279/not-able-to-get-tensorflow-go ) [link](https://github.com/tensorflow/tensorflow/issues/13765) #3437

**Comment by lolbinarycat** [comment]
```
cannot find package "github.com/tensorflow/tensorflow/tensorflow/go/core/protobuf/for_core_protos_go_proto" in any of:
        /usr/lib/go-1.15/src/github.com/tensorflow/tensorflow/tensorflow/go/core/protobuf/for_core_protos_go_proto (from $GOROOT)
        /home/binarycat/go/src/github.com/tensorflow/tensorflow/tensorflow/go/core/protobuf/for_core_protos_go_proto (from $GOPATH)
```

**Comment by Saduf2019** [comment]
@lolbinarycat 
Can you please refer to this issue and let us know if it helps. #42657

**Comment by lolbinarycat** [comment]
> @lolbinarycat
> Can you please refer to this issue and let us know if it helps. #42657

I was reading through it, but It's hard to follow along as running a single `bazel` command takes around a day, and it makes my computer slow. If there are any steps I should take in particular let me know, but I didn't see anything that looked super helpful.

**Comment by nyeinsoe26** [comment]
let me tell you the steps I took to resolve this:
1. Install TensorFlow C based on the instructions given in this website: ->          https://www.tensorflow.org/install/lang_c
2. Once tensorflow C is installed, install go:
            go get -v github.com/tensorflow/tensorflow/tensorflow/go
3. If it succeeds by this step, then grats. if not run this command:
            cd $GOPATH/src/github.com/tensorflow/tensorflow/tensorflow/go
            git checkout tags/v2.1.0
4. Test your build with 
            go test github.com/tensorflow/tensorflow/tensorflow/go
    if the test produce this line "ok github.com/tensorflow/tensorflow/tensorflow/go 0.152s" at the end, then you are good to go. At least it did for me. Good luck

**Comment by lolbinarycat** [comment]
Thanks! I'll try that when I have the time.
